### PR TITLE
fix 'NoneType' object has no attribute 'utcoffset'

### PR DESCRIPTION
**Pull request by paulfab**
(Describe the goal of this PR. Mention any related Issue numbers)

### Category (place an `x` in each of the `[ ]`)

* [ ] `slack_bolt.App` and/or its core components
* [ ] `slack_bolt.async_app.AsyncApp` and/or its core components
* [ ] Document pages under `/docs`
* [ ] Adapters in `slack_bolt.adapter`
* [ x] Others

## Requirements (place an `x` in each `[ ]`)

Please read the [Contributing guidelines](https://github.com/slackapi/bolt-python/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to those rules.

* [ x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/bolt-python/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x ] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [ ] I've run `./scripts/install_all_and_run_tests.sh` after making the changes.

### CODE REVIEW

**Comment by seratch** [comment]
Hi @paulfab, thanks for taking the time to fix this error! Would you mind signing the CLA (please check the above bot message)? Without having it, we are unable to merge your changes in this repository.

File: examples/django/oauth_app/slack_datastores.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def save_bot(self, bot: Bot):
         b = bot.to_dict()
         if is_naive(b["installed_at"]):
             b["installed_at"] = make_aware(b["installed_at"])
-        if "bot_token_expires_at" in b is not None and is_naive(
```
Comment (seratch) [inline examples/django/oauth_app/slack_datastores.py:61]: Obviously, this part was wrong.

File: examples/django/oauth_app/slack_datastores.py [hunk 2]
```diff
@@ -33,9 +33,9 @@ def save(self, installation: Installation):
         i = installation.to_dict()
         if is_naive(i["installed_at"]):
             i["installed_at"] = make_aware(i["installed_at"])
-        if "bot_token_expires_at" in i and is_naive(i["bot_token_expires_at"]):
+        if i.get("bot_token_expires_at") and is_naive(i["bot_token_expires_at"]):
```
Comment (seratch) [inline examples/django/oauth_app/slack_datastores.py:36]: Can you change this way for consistency?
```suggestion
        if i.get("bot_token_expires_at") is not None and is_naive(i["bot_token_expires_at"]):
```

File: examples/django/oauth_app/slack_datastores.py [hunk 3]
```diff
@@ -33,9 +33,9 @@ def save(self, installation: Installation):
...
             i["bot_token_expires_at"] = make_aware(i["bot_token_expires_at"])
-        if "user_token_expires_at" in i and is_naive(i["user_token_expires_at"]):
+        if i.get("user_token_expires_at") and is_naive(i["user_token_expires_at"]):
```
Comment (seratch) [inline examples/django/oauth_app/slack_datastores.py:38]: same as above
```suggestion
        if i.get("user_token_expires_at") is not None and is_naive(i["user_token_expires_at"]):
```

**Review by seratch — CHANGES_REQUESTED** [review]

**Comment by seratch** [comment]
@paulfab Thanks for updating this pull request. Would you mind signing the CLA (please check the above bot message)? Without having it, we are unable to merge your changes in this repository.

**Comment by paulfab** [comment]
Hey @seratch you are so fast I didnt have the time to comment ^^. Thanks for reviewing the PR, I'm quite reluctant to sign the CLA. You can take ownership of my modest changes and merge it ?

**Comment by seratch** [comment]
@paulfab Okay, in the case, we can come up with my own pull request instead. Thanks a lot for taking the time to report this!

**Comment by seratch** [comment]
Let me close this pull request in favor of https://github.com/slackapi/bolt-python/pull/523

**Pull request closed without merging**